cudax/include/cuda/experimental/__async/sender/just_from.cuh

```diff
@@ -93,7 +93,7 @@ private:
   };
 
   template <class _Rcvr, class _Fn>
-  struct __opstate
+  struct _CCCL_TYPE_VISIBILITY_DEFAULT __opstate
```

Comment:
I would appreciate if we could factor out the cleanup from the functional changes and move them to a separate PR

Reply:
i have created pr #4561 for these cleanups. i will rebase this pr on that once it lands.